src/components/Footer/Footer.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './Footer.scss';
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="top">
+        <div className="container">
+          <ul className="footer_menu">
+            <li>인재채용</li>
+            <li>이용약관</li>
+            <li className="personal_info">개인정보처리방침</li>
+          </ul>
+          <button className="app_btn">정육쩜 앱 설치</button>
+        </div>
+      </div>
+      <div className="bottom">
+        <div className="container">
+          <div className="corp_info">
+            <h1 className="ft_logo">
+              <img src="./images/logo_b.png" alt="푸터로고" />
```

Comment:
img src 경로로는 상대경로가 아닌 절대경로로 작성하여 public 폴더로 접근할 수 있습니다.
```suggestion
              <img src="/images/logo_b.png" alt="푸터로고" />
```